website/docs/Getting-Started.mdx

```diff
@@ -112,27 +110,33 @@ Open `coding/plot.png` to see the generated plot.
 Learn more about configuring LLMs for agents [here](/docs/topics/llm_configuration).
 :::
 
-
-
 #### Multi-Agent Conversation Framework
+
 Autogen enables the next-gen LLM applications with a generic multi-agent conversation framework. It offers customizable and conversable agents which integrate LLMs, tools, and humans.
 By automating chat among multiple capable agents, one can easily make them collectively perform tasks autonomously or with human feedback, including tasks that require using tools via code. For [example](https://github.com/microsoft/autogen/blob/main/test/twoagent.py),
 
 The figure below shows an example conversation flow with AutoGen.
 
 ![Agent Chat Example](/img/chat_example.png)
 
-
 ### Where to Go Next?
 
-* Go through the [tutorial](/docs/tutorial/introduction) to learn more about the core concepts in AutoGen
-* Read the examples and guides in the [notebooks section](/docs/notebooks)
-* Understand the use cases for [multi-agent conversation](/docs/Use-Cases/agent_chat) and [enhanced LLM inference](/docs/Use-Cases/enhanced_inference)
-* Read the [API](/docs/reference/agentchat/conversable_agent/) docs
-* Learn about [research](/docs/Research) around AutoGen
-* Chat on [Discord](https://discord.gg/pAbnFJrkgZ)
-* Follow on [Twitter](https://twitter.com/pyautogen)
+- Go through the [tutorial](/docs/tutorial/introduction) to learn more about the core concepts in AutoGen
+- Read the examples and guides in the [notebooks section](/docs/notebooks)
+- See our [roadmaps](https://github.com/microsoft/autogen/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap)
```

Comment:
Move to the end of the list?

Reply:
Agreed this would be good as the final item